src/Tizen.NUI.Components/Controls/Notification.cs

```diff
@@ -0,0 +1,394 @@
+                    notificationWindow.Show();
+                }
+
+                return notificationWindow;
+            }
+            set => notificationWindow = value;
+        }
+
+        private Timer Timer
+        {
+            get
+            {
+                if (timer == null)
+                {
+                    timer = new Timer(0);
+                    timer.Tick += OnTimeOut;
+                }
+
+                return timer;
+            }
+            set
+            {
+                if (timer != null && timer.IsRunning())
+                {
+                    timer.Stop();
+                }
+
+                timer = value;
+            }
+        }
+
+        private Timer ExtraTimer
+        {
+            get
+            {
+                if (extraTimer == null)
+                {
+                    extraTimer = new Timer(0);
+                    extraTimer.Tick += OnExtraTimeOut;
+                }
+
+                return extraTimer;
+            }
+            set
+            {
+                if (extraTimer != null && extraTimer.IsRunning())
+                {
+                    extraTimer.Stop();
+                }
+
+                extraTimer = value;
+            }
+        }
+
+        /// <summary>
+        /// Post a notification window with the content view.
+        /// </summary>
+        /// <param name="duration">Dismiss the notification window after given time. The value 0 won't dismiss the notification.</param>
+        /// <returns>The current Notification instance.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Notification Post(uint duration = 0)
+        {
+            if (state != NotificationState.Ready)
+            {
+                return this;
+            }
+
+            var window = NotificationWindow;
+
+            ApplyLevel(level);
+
+            ApplyPositionSize(positionSize);
+
+            ApplyDismissOnTouch(dismissOnTouch);
+
+            NotificationWindow.Add(ContentView);
+
+            if (duration > 0)
+            {
+                Timer.Interval = duration;
+            }
+
+            state = NotificationState.Post;
+
+            if (onPost != null)
+            {
+                onPost(this);
+            }
+
+            return this;
+        }
+
+        /// <summary>
+        /// Sets a priority level for the specified notification window.
+        /// </summary>
+        /// <param name="level">The notification window level.</param>
+        /// <returns>The current Notification instance.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Notification SetLevel(NotificationLevel level)
+        {
+            this.level = level;
+
+            if (state == NotificationState.Post || state == NotificationState.DelayingOnDismiss)
+            {
+                ApplyLevel(level);
+            }
+
+            return this;
```

Comment:
Why Notification instance is returned after setting level?

Reply:
To enable calling API continuously.
```C#
new Notification(contentView)
    .SetDismissOnTouch(true)
    .SetOnPostDelegate(OnNotificationPost)
    .SetOnDismissDelegate(OnNotificationDismiss)
    .Post();
```